clash-lib/src/Clash/Netlist.hs

```diff
@@ -322,6 +323,7 @@ mkNetDecl (id_,tm) = preserveVarEnv $ do
 
   where
     multiDecls pInfo args0 = do
+      san <- asks _sanitizeNames
```

Comment:
We should be consistent and use `Lens.view sanitizeNames` instead of `asks _sanitizeNames`. I don't particularly like lenses, but this is the style we tend to use here so let's stick with it

Reply:
Implemented!